src/coreclr/vm/codeman.cpp

```diff
@@ -3850,6 +3860,18 @@ TADDR EEJitManager::FindMethodCode(RangeSection * pRangeSection, PCODE currentPC
 }
 
 #if !defined(DACCESS_COMPILE)
+
+void EEJitManager::NibbleMapSetUnlocked(HeapList * pHp, TADDR pCode, BOOL bSet)
```

Comment:
We typically use `Unblocked` to mean that the method does not take any locks, and that the locks are taken by caller or that the thread safefy is guaranteed by other means. It is used in exact opposite meaning here.

Reply:
I was looking at EntryPointSlots::Backpatch_Locked, MethodDescBackpatchInfoTracker::AddSlotAndPatch_Locked and few others which use "Locked" with the meaning that it needs to be called under a lock. 
I've now swapped the NibbleMapSetUnlocked and NibbleMapSet.